docs/operate/config-reference.mdx

```diff
@@ -1075,6 +1075,50 @@ Because each VM supports a different signer type, the default signer can only su
 </TabItem>
 </Tabs>
 
+## defaultRpcConsensusType
+
+:::info
+
+**The consensus type currently only applies to EVM chains.**
+
+Connections to non-EVM chains only support a single RPC url. If more than one RPC url is specified in [rpcUrls](#chainschain_namerpcurls), only the first one will be used.
+
+:::
+
+**Description:** How to communicate with the provider
+
+- `"single"` uses a basic http connection
+- `"fallback"` will try the first URL and then automatically "fall back" on a connection or other provider failure to the next provider; this should only retry calls which fail due to the provider and not the call itself such; an insufficient gas error for instance would not fall back.
+- `"quorum"` requires a majority of the URLs to agree with the exception of submitting transactions; it will automatically coordinate the "latest" block if not specified to reduce sync errors.
+
+**Optional:** Defaults to `"fallback"`
+
+**Agents:** All
+
+**Type:** `Enum ("fallback", "single", "quorum")`
+
+<Tabs groupId="config-reference">
+<TabItem value="arg" label="As Arg" default>
+```bash
+--defaultRpcConsensusType single
+```
+</TabItem>
+
+<TabItem value="env" label="As Env">
+  ```bash
+  export HYP_DEFAULT_RPCCONSENSUSTYPE="single"
```

Comment:
```suggestion
  export HYP_DEFAULTRPCCONSENSUSTYPE="single"
```

Reply:
renamed in 5849a4ef699a1fb274737762445f8518818cc383